### ISSUE TITLE
issue template missing information for upgrade job

### ISSUE THREAD

**Issue by fgelcer** [scylladb/argus#212]
for upgrade jobs we have both `base version` and `target version` fields, but in the issue template, there is only `scylla_version`:

```
Run Details
Release: enterprise-2023.1
Group: Cluster Rolling Upgrades
Test: rolling-upgrade-centos8-test
Id: 6de6634c-2264-4834-b324-6be7f2a59b5c
Start time: 2023-03-02 12:31:57
End time: 2023-03-02 19:44:09
Duration: 7 hours
Started by: yaronkaikov
Build job: enterprise-2023.1/rolling-upgrade/rolling-upgrade-centos8-test
System Information
Backend: gce
Region: us-east1
Image id: https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-stream-8
Kernel version: 4.18.0-448.el8.x86_64
Base scylla version: 5.2.0~rc1-20230207.8ff4717fd010
Build id: 78fbb2c25e9244a62f57988313388a0260084528
Upgraded scylla version: 2023.1.0~rc2-20230302.726f8a090337
Build id: efcc950cda22875aa488c5295c6cc68f35d9cc6f
Instance type: n1-highmem-8
Node amount: 4
```

```

## Issue description

- [ ] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

*Describe your issue in detail and steps it took to produce it.*

## Impact

*Describe the impact this issue causes to the user.*

## How frequently does it reproduce?

*Describe the frequency with how this issue can be reproduced.*

## Installation details

Kernel Version: 4.18.0-448.el8.x86_64
Scylla version (or git commit hash): `5.2.0~rc1-20230207.8ff4717fd010` with build-id `78fbb2c25e9244a62f57988313388a0260084528`




Cluster size: 4 nodes (n1-highmem-8)

Scylla Nodes used in this run:
- rolling-upgrade--centos-8-db-node-6de6634c-0-4 (35.231.169.247 | 10.142.0.225) (shards: 8)
- rolling-upgrade--centos-8-db-node-6de6634c-0-3 (34.148.4.0 | 10.142.0.218) (shards: 8)
- rolling-upgrade--centos-8-db-node-6de6634c-0-2 (34.138.31.167 | 10.142.0.215) (shards: 8)
- rolling-upgrade--centos-8-db-node-6de6634c-0-1 (35.196.193.45 | 10.142.0.209) (shards: 8)


OS / Image: `https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-stream-8` (gce: us-east1)

Test: `rolling-upgrade-centos8-test`
Test id: `6de6634c-2264-4834-b324-6be7f2a59b5c`
Test name: `enterprise-2023.1/rolling-upgrade/rolling-upgrade-centos8-test`
Test config file(s):

- [rolling-upgrade.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/9a5f163455e77d1518d00ea2a5ac14df134cf2f6/test-cases/upgrades/rolling-upgrade.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor 6de6634c-2264-4834-b324-6be7f2a59b5c`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=6de6634c-2264-4834-b324-6be7f2a59b5c)
- Show all stored logs command: `$ hydra investigate show-logs 6de6634c-2264-4834-b324-6be7f2a59b5c`


## Logs:
*No logs captured during this run.*

[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2023.1/job/rolling-upgrade/job/rolling-upgrade-centos8-test/7/)
</details>
```

**Comment by fgelcer** [comment]
@ShlomiBalalis should have used `Fixes: #212` instead of Solves...
hence closing this one as done

**Comment by ShlomiBalalis** [comment]
> @ShlomiBalalis should have used `Fixes: #212` instead of Solves... hence closing this one as done

Will keep that in mind
